pkgs/top-level/python-packages.nix

```
@@ -5608,6 +5608,10 @@ in {
 
   safe = callPackage ../development/python-modules/safe { };
 
+  sane = callPackage ../development/python-modules/sane {
```

this should probably match the pypi name, so it feels similar to a typical setup.py or requirements.txt
```suggestion
  python-sane = callPackage ../development/python-modules/python-sane {
```

cc @FRidh thoughts?

---

I think they named the project `python-sane` to differentiate it from what we call `saneBackends`. They name themselves just "Sane" on GitHub: https://github.com/python-pillow/Sane

---

Last push is to mark it as python3 only.